tests/renderStatsCard.test.js

```diff
@@ -75,6 +75,22 @@ describe("Test renderStatsCard", () => {
     expect(queryByTestId(document.body, "rank-circle")).not.toBeInTheDocument();
   });
 
+  it("should render with custom width set", () => {
+    document.body.innerHTML = renderStatsCard(stats);
+    expect(document.querySelector("svg")).toHaveAttribute("width", "495");
+
+    document.body.innerHTML = renderStatsCard(stats, { card_width: 400 });
+    expect(document.querySelector("svg")).toHaveAttribute("width", "400");
+  });
+
+  it("should render with custom width set and limit minimum width", () => {
+    document.body.innerHTML = renderStatsCard(stats, { card_width: 1 });
+    expect(document.querySelector("svg")).toHaveAttribute("width", "340");
+
+    document.body.innerHTML = renderStatsCard(stats, { card_width: 1, hide_rank: true });
```

Comment:
@postatum Maybe we should also add tests for the `show_icons` option. My commit now adds `16px` when this option is enabled.

![Rick's Github Stats](https://github-readme-stats-taupe-pi.vercel.app/api?username=rickstaa&card_width=1&hide_rank=true&show_icons=true)